### ISSUE TITLE
1.2.0: Call to undefined method Magenerds\BasePrice\Model\Plugin\AfterPrice::_getAfterPriceHtml()

### ISSUE THREAD

**Issue by damac23** [Magenerds/BasePrice#26]
That's the current error message:
```
[08-Feb-2018 07:20:37 UTC] PHP Fatal error:  Uncaught Error: Call to undefined method Magenerds\BasePrice\Model\Plugin\AfterPrice::_getAfterPriceHtml() in /var/www/MYSHOP/www.MYSHOP/htdocs/vendor/magenerds/baseprice/Model/Plugin/AfterPrice.php:84
Stack trace:
#0 /var/www/MYSHOP/www.MYSHOP/htdocs/vendor/magento/framework/Interception/Interceptor.php(135): Magenerds\BasePrice\Model\Plugin\AfterPrice->aroundRender(Object(Magento\Framework\Pricing\Render\Interceptor), Object(Closure), 'final_price', Object(Magento\Catalog\Model\Product\Interceptor), Array)
#1 /var/www/MYSHOP/www.MYSHOP/htdocs/vendor/magento/framework/Interception/Interceptor.php(153): Magento\Framework\Pricing\Render\Interceptor->Magento\Framework\Interception\{closure}('final_price', Object(Magento\Catalog\Model\Product\Interceptor), Array)
#2 /var/www/MYSHOP/www.MYSHOP/htdocs/generated/code/Magento/Framework/Pricing/Render/Interceptor.php(26): Magento\Framework\Pricing\Render\Interceptor->___callPlugins('rende in /var/www/MYSHOP/www.MYSHOP/htdocs/vendor/magenerds/baseprice/Model/Plugin/AfterPrice.php on line 84
```

The _getAfterPriceHtml() is now called getAfterPriceHtml(), but does require a SaleableInterface.

**Comment by sydekumf** [comment]
Thanks! Fixed by: https://github.com/Magenerds/BasePrice/releases/tag/1.2.1

**Comment by damac23** [comment]
Thanks for the fast fix, but it's not fully finished :)

this
`$this->getAfterPriceHtml(productInterceptor)`
does not work, but this does:
`$this->getAfterPriceHtml($productInterceptor)`

**Comment by sydekumf** [comment]
Yeah... embarassing :-( Fixed it now in 1.2.1 :-)
